dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/DatesBindingTest.java

```diff
@@ -54,10 +54,7 @@ class DatesBindingTest {
 
   @BeforeEach
   public void setUp() {
-    mockMvc =
-        MockMvcBuilders.standaloneSetup(new BindingController())
-            .setControllerAdvice(new CrudControllerAdvice())
-            .build();
+    mockMvc = MockMvcBuilders.standaloneSetup(new BindingController()).build();
```

Comment:
Can you adopt asserting like we do in dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/OrderBindingTest.java. So instead of asserting on strings in the response body use the actual type.

Also we replaced the word `Criteria` with `RequestParams` in the web layer so I would not add new code even test code using the word `criteria`.